server/src/main/scala/net/wiringbits/tasks/NotificationsTask.scala

```diff
@@ -0,0 +1,50 @@
+package net.wiringbits.tasks
+
+import akka.actor.ActorSystem
+import com.google.inject.Inject
+import net.wiringbits.actions.internal.{GetPendingNotificationsAction, SendNotificationAction}
+import net.wiringbits.config.NotificationsConfig
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+class NotificationsTask @Inject() (
+    notificationsConfig: NotificationsConfig,
+    getPendingNotifications: GetPendingNotificationsAction,
+    sendNotificationAction: SendNotificationAction
+)(implicit
+    ec: ExecutionContext,
+    actorSystem: ActorSystem
+) {
+  val logger = LoggerFactory.getLogger(this.getClass)
+
+  logger.info("Starting the notifications task")
+  actorSystem.scheduler.scheduleOnce(
+    notificationsConfig.interval
+  ) {
+    run()
+  }
+
+  def run(): Unit = {
+    logger.info("Looking for notifications")
```

Comment:
This is good for debugging but noisy when everything works, either remove it or use `logger.trace`